README.md

```diff
@@ -1,14 +1,14 @@
 # Supported Windows Server 2016 Version 1709 (Fall Creators Update) amd64 tags
 
-- [`4.7.1-windowsservercore-1709` (*4.7.1-windowsservercore-1709/Dockerfile*)](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.7.1-windowsservercore-1709/Dockerfile)
-- [`3.5-windowsservercore-1709` (*3.5-windowsservercore-1709/Dockerfile*)](https://github.com/Microsoft/dotnet-framework-docker/blob/master/3.5-windowsservercore-1709/Dockerfile)
+- [`4.7.1-windowsservercore-1709`, `4.7.1`, `latest` (*4.7.1-windowsservercore-1709/Dockerfile*)](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.7.1-windowsservercore-1709/Dockerfile)
```

Comment:
Were we going to make note that these manifest tags require latest Docker?

Reply:
Good call. I made the following update to the [.NET Core announcement](https://github.com/dotnet/dotnet-docker/issues/330). If you like that text ... I'll do the same update for .NET Framework and then add appropriate links in each README. Tell me if that works. Feel free to update the text directly.

This paragraph:

> .NET Core [multi-arch tags](https://github.com/dotnet/announcements/issues/14), such as `2.0-runtime`, have been updated to use `nanoserver-1709` images if your host is Windows Server 1709 or higher or Windows 10 Fall Creators Update (Version 1709) or higher. You need [Docker 17.10](https://docs.docker.com/release-notes/docker-ce/#17100-ce-2017-10-17) or later to take advantage of these updated tags.

Reply:
LGTM